### PR TITLE
Terraformed ETL lambda

### LINKED ISSUE #6: Terraform Lambda for ETL pipeline
**Description:**
Terraform code to create an AWS Lambda.

**Outcome:**
AWS Lambda can be created from code, to run the ETL pipeline via its ECR image URI.

**User Story:**
As a developer, I want to automate the process of creating a lambda on AWS so that I can maintain and repeat this process if need be.

**Design:**
- [x] lambda terraform code
- [x] security groups 
- [x] logging 
- [x] Event bridge schedule for everyday every minute

### PR DESCRIPTION
**Pull request by FarihaChoudhury**
Created lambda with terraform code to ensure repeatability/reusability. Also created event bridge to schedule lambda for every minute. 
Updated readme with features. 

Closes #6

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  id = "subnet-0497831b67192adc2"
+}
+
+data "aws_subnet" "c14-subnet-2" {
+  id = "subnet-0acda1bd2efbf3922"
+}
+
+data "aws_subnet" "c14-subnet-3" {
+  id = "subnet-0465f224c7432a02e"
+}
```
Comment (qrafiq1) [inline terraform/main.tf:21]: make variables for future use?

File: terraform/README.MD [hunk 2]
```diff
@@ -1,7 +1,61 @@
...
+
+DB_HOST=""
+DB_PORT=""
+DB_PASSWORD=""
+DB_USER=""
+DB_NAME=""
+SCHEMA_NAME=""
+
+ETL_ECR_URI=""
+```
```
Comment (SurinaCS) [inline terraform/README.MD:61]: can you make this into a table using a description as another column:
| Variable         | Description                                       |
|------------------|---------------------------------------------------|
| AWS_ACCESS_KEY   | The access key for AWS authentication.           |
| AWS_SECRET_KEY   | The secret key for AWS authentication.           |
| C14_VPC          | The VPC (Virtual Private Cloud) ID.              |
| DB_HOST          | The hostname or IP address of the database.      |
| DB_PORT          | The port number for the database connection.     |
| DB_PASSWORD      | The password for the database user.              |
| DB_USER          | The username for the database.                   |
| DB_NAME          | The name of the database.                        |
| SCHEMA_NAME      | The name of the database schema.                 |
| ETL_ECR_URI      | The URI for the ETL container repository in ECR. |

**Review by qrafiq1 — CHANGES_REQUESTED** [review]

**Review by SurinaCS — COMMENTED** [review]
nice

File: terraform/README.MD (hunk 2 above)
Comment (FarihaChoudhury) [reply terraform/README.MD:61]:
```suggestion
| Variable         | Description                                       |
|------------------|---------------------------------------------------|
| AWS_ACCESS_KEY   | The access key for AWS authentication.           |
| AWS_SECRET_KEY   | The secret key for AWS authentication.           |
| C14_VPC          | The VPC (Virtual Private Cloud) ID.              |
| DB_HOST          | The hostname or IP address of the database.      |
| DB_PORT          | The port number for the database connection.     |
| DB_PASSWORD      | The password for the database user.              |
| DB_USER          | The username for the database.                   |
| DB_NAME          | The name of the database.                        |
| SCHEMA_NAME      | The name of the database schema.                 |
| ETL_ECR_URI      | The URI for the ETL container repository in ECR. |
```

**Review by qrafiq1 — APPROVED** [review]
Nioce

**Pull request merged**